microservices/time-series-analytics/third-party-programs.txt

```diff
@@ -17,77 +17,81 @@ terms are listed below.
 ia-time-series-analytics-microservice:
 ======================================
 
-1. Base docker image: kapacitor:1.7.7
+1. Base docker image: kapacitor:1.8.2
 License-Identifier: MIT License
 
 2. requirements.txt
-License-Identifier: Apache License 2.0
 
-anyio==4.9.0
-fastapi==0.115.12
-fsspec==2025.5.1
+License-Identifier: Apache License 2.0
+dpctl==0.20.2
+fastapi==0.121.0
 influxdb==5.3.2
-joblib==1.5.1
-modin==0.32.0
+pyOpenSSL==25.3.0
+joblib==1.5.2
 numpy==2.2.6
-packaging==25.0
-pandas==2.2.3
-protobuf==4.24.4
-psutil==7.0.0
-pydantic==2.11.5
-pydantic_core==2.33.2
+protobuf==6.31.1
+pydantic==2.12.4
+pydantic_core==2.41.5
 python-dateutil==2.9.0.post0
-scikit-learn==1.6.1
 scikit-learn-intelex==2025.2.0
 scipy==1.15.3
 sortedcontainers==2.4.0
-starlette==0.46.2
+starlette==0.49.3
 threadpoolctl==3.6.0
 tomlkit==0.13.2
-typing_extensions==4.14.0
-typing-inspection==0.4.1
-tzdata==2025.2
+typing-inspection==0.4.2
+typing_extensions==4.15.0
 uvicorn==0.34.0
-watchdog==6.0.0
+anyio==4.11.0
+fastapi==0.115.12
 
 License-Identifier: MIT License
-
-aiofiles==24.1.0
-aiosqlite==0.21.0
-annotated-types==0.7.0
-certifi==2025.4.26
-charset-normalizer==3.4.2
-click==8.2.1
+click==8.3.1
 exceptiongroup==1.3.0
 h11==0.16.0
-idna==3.10
-msgpack==1.1.0
-requests==2.32.3
+aiofiles==25.1.0
+aiosqlite==0.21.0
+annotated-types==0.7.0
+certifi==2025.11.12
+idna==3.11
+msgpack==1.1.2
+requests==2.32.5
 six==1.17.0
 sniffio==1.3.1
-wheel==0.37.1
+urllib3==2.5.0
+charset-normalizer==3.4.4
+annotated-doc==0.0.4
+cffi==2.0.0
 
 License-Identifier: BSD License
 
-daal==2025.2.0
-tbb==2022.1.0
-pycparser==2.22
-pandas==2.2.3
+pycparser==2.23
+pytz==2025.2
 scikit-learn==1.6.1
+tbb==2022.3.0
+tcmlib==1.4.1
+daal==2025.7.0
 scipy==1.15.3
-pytz==2025.2
 
 License-Identifier: LGPL-3.0
 
 asyncua==1.1.5
 
 License-Identifier: Apache-2.0 or BSD-3-Clause

```

Comment:
Let's add one line below each license category like --------------------------- to group the packages having common licenses